bindings/jupyroot/python/JupyROOT/helpers/utils.py

```diff
@@ -58,6 +58,7 @@
  require(['JSRootCore'],
      function(Core) {{
        var obj = Core.JSONR_unref({jsonContent});
+       Core.key_handling = false;
```

Comment:
Thanks @linev , is this something we need to do every time we draw, or we could do it once when we enable JSRoot in the notebook?

Reply:
It can be done once after first loading of `JSRootCore` module. 
But it does not make any big difference - it is just global JSROOT settings and can be configured at any time before next `JSROOT.draw()` call

Reply:
Would it be possible/easy to do it once after loading `JSRootCore`? I understand the effect is the same, but it is just not to have the assignment done every time we draw if once is enough.

Reply:
But `JSRootCore` module loaded only immediately before first draw - it does not exists before.